jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.java

```diff
@@ -61,5 +56,7 @@ private void createUIComponents() {
         this.showLogGroupTable = new AddRemoveLogGroupTable(project);
         initArLogGroupTable();
         tablePanel.setContent(showLogGroupTable.getComponent());
+        String[] TimeUnits = new String[] {"Minutes", "Hours", "Days", "Weeks"};
+        relativeTimeUnit = new ComboBox(TimeUnits);
```

Comment:
You can use `arrayOf(...)` instead

Reply:
Converted Java file to Kotlin. Changed it to arrayOf()